tests/unit/test_gitlab.py

```diff
@@ -18,6 +18,7 @@
 
 import pickle
 import warnings
+from copy import deepcopy
```

Comment:
nit: My personal taste is `import copy` and then use `copy.deepcopy()` as when reading the code quickly know where it comes from.